Example_for_code_review.txt

```diff
@@ -0,0 +1,19 @@
+##
+Hi!
+
+this is just a simple text file for us to learn abput pull request - or as we prefer to short thier name - PR
+
+so
+
+lest say we have here a segment of code:
+
+
+nirs_method(int iterations){
+	for(int i=0 ; i < iterations ; i++) {
+		x=5;
```

Comment:
x = 30

Reply:
cool, so, this is just a bit of a feel of how we reviewing. it's a conversation :)
after we done reviewing, if you would approve all my changes, Ill be able in PR amin screen to 'merge to master'. which mean, take all the commits I added here and put them in the main branch (master).
try create PR and I'll review